src/gt4py/next/ffront/stages.py

```diff
@@ -148,4 +153,3 @@ def add_foast_located_node_to_fingerprint(
 ) -> None:
     add_content_to_fingerprint(obj.location, hasher)
     add_content_to_fingerprint(str(obj), hasher)
-    add_content_to_fingerprint(str(obj), hasher)
```

Comment:
I am not sure, why this was done twice, does removing it make sense @egparedes?

Reply:
It looks like a bug to me and I agree with your change.